spaces/S000185/properties/P000061.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000185
+property: P000061
+value: true
+---
+
+As singletons from $a \in \omega^2$ are connected components, $\{a\}$ is cozero.
+So as countable unions of cozero sets are cozero, every subset of $\omega^2$ is a cozero set.
+
+Thus, given a cozero set $A$, $B := X \setminus (A \cup \{\infty_1,\infty_2\})$ is cozero and $\omega^2 \subseteq A \cup B$, which is dense.
```

Comment:
```suggestion
Singletons $\{a\}$ with $a\in\omega^2$ are clopen sets, hence cozero sets.
So every subset of $\omega^2$ is a cozero set, as a countable union of cozero sets.

Thus, given a cozero set $A$, the set $B := X \setminus (A \cup \{\infty_1,\infty_2\})$ is a cozero set disjoint from $A$, and $A \cup B$ is dense in $X$ since it contains $\omega^2$.
```
rewording for readability